canonical/store-components
# Add footer to PackageCard component

## Done
- Added a footer for cards with a platform or libraries link

## QA
- Go to https://store-components-11.demos.haus/?path=/story/packagecard--platform
- Check that there is a footer with the platform logo
- Go to https://store-components-11.demos.haus/?path=/story/packagecard--interface-story
- Check that there is a footer with a link to "Libraries"

## Issues
- Fixes https://warthogs.atlassian.net/browse/WD-1625
- Fixes https://warthogs.atlassian.net/browse/WD-1626
